modules/shortcodes/polldaddy.php

```diff
@@ -573,7 +573,7 @@ function polldaddy_link( $content ) {
 	add_filter( 'the_content_rss', 'polldaddy_link', 1 );
 
 	/** This filter is documented in modules/shortcodes/youtube.php */
-	if ( apply_filters( 'jetpack_comments_allow_oembed', get_option( 'embed_autourls' ) ) ) {
+	if ( apply_filters( 'jetpack_comments_allow_oembed', true ) ) {
```

Comment:
I think we should stick with the `get_option( 'embed_autourls' )` call. The option may no longer exist, but people could still be attaching things to the `default_option_embed_autourls` filter (which would be a bit silly, but it's possible).